src/cryptoadvance/specter/wallet.py

```diff
@@ -845,3 +873,26 @@ def importpsbt(self, b64psbt):
             psbt["raw"] = raw["hex"]
         self.save_pending_psbt(psbt)
         return psbt
+
+    @property
+    def weight_per_input(self):
+        """Calculates the weight of a signed input"""
+        if self.is_multisig:
+            input_size = 3 # OP_M OP_N ... OP_CHECKMULTISIG
+            for i in range(0, len(self.keys)):
+                # pubkey size
+                input_size += 34
+            for i in range(0, self.sigs_required):
+                input_size += 75 # max sig size
+
+            if not self.recv_descriptor.startswith('wsh'):
+                # P2SH scriptsig: 00 20 <32-byte-hash>
+                input_size += 34 * 4
+            return input_size
+        else:
+            if self.recv_descriptor.startswith('wpkh'):
+                # pubkey, signature
+                return 75 + 34
+            else:
+                # pubkey, signature, 4* P2SH: 00 14 20-byte-hash
+                return 75 + 34 + 22 * 4
```

Comment:
The `else` statement could be removed here, right? This would avoid the unnecessary indent.